compiler/noirc_evaluator/src/ssa/opt/flatten_cfg/capacity_tracker.rs

```diff
@@ -72,7 +72,6 @@ impl<'a> SliceCapacityTracker<'a> {
                         // where v7 is the slice length and v8 is the popped slice itself.
                         Intrinsic::SlicePopFront => (Some(1), results.len() - 1),
                         // The slice capacity of these intrinsics is not determined by the arguments of the function.
```

Comment:
```suggestion
```
This comment can technically be deleted. We can also remove the option wrapper around `argument_index`. This is a nit though and we can do it in a follow-up